src/metabase/models/setting.clj

```diff
@@ -1137,15 +1165,14 @@
   in [[metabase.server.routes.index/load-entrypoint-template]]. These are used as read-only sources of Settings for
   the frontend client. For that reason, these Settings *should* include values that come back from environment
   variables, *unless* they are marked `:sensitive?`."
-  [visibility]
+  [visibilities]
```

Comment:
This function was usually called multiple times in one place, iterating over a set of visibilities, then merging the results of each call.

e.g. [here](https://github.com/metabase/metabase/pull/28150/files#diff-d0612df4ce6645f61fe0f1ad4d24a0c44abc5c6f42afab8f423f22ecfd732c05L282)

To simplify its use I've changed its argument to be a set of visibilities.